assets/styl/_elements/_pager.styl

```diff
@@ -45,6 +46,17 @@
             width 1em
             height @width
 
+        // this after pseudo element extend clickable area
+        // when the button is too much small
+        &:after
```

Comment:
Shouldn't this appears only on mobile ?

Reply:
Click area can be bigger on all screens, and a desktop can be tactile